src/test/java/address/TestApp.java

```diff
@@ -56,6 +60,7 @@ protected Config initConfig(String configFilePath) {
     @Override
     protected UserPrefs initPrefs(Config config) {
         UserPrefs userPrefs = super.initPrefs(config);
+        userPrefs.setGuiSettings(new GuiSettings(600.0, 600.0, 0, 0));
```

Comment:
I think this may not be a good solution. Positioning the window at 0, 0 will also cause the headful tests to fail on Mac, since the top task-bar (whatever that's called) will cover the app UI elements.